qt/tests_backend_qt/widgets/button.py

```diff
@@ -0,0 +1,24 @@
+from PySide6.QtWidgets import QPushButton
+
+from .base import SimpleProbe
+
+
+class ButtonProbe(SimpleProbe):
+    native_class = QPushButton
+
+    @property
+    def text(self):
+        # Normalize the zero width space to the empty string.
+        if self.native.text() == "\u200b":
+            return ""
+        return self.native.text()
+
+    def assert_no_icon(self):
+        assert self.native.icon().isNull()
+
+    def assert_icon_size(self):
+        # No assertion needed here.  It is handled by theme.
+        # [better not write anything more here just in case of
+        # anything about size accidentally being an inappropriate
+        # joke]
```

Comment:
```suggestion
        # Icons sizes in Qt are handled by the system theme;
        # no assertion is needed here.
```

Learnt by lesson after being told to not make a specific joke...